staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -242,14 +257,27 @@ func (m *AdmissionMetrics) ObserveAdmissionController(ctx context.Context, elaps
 	m.controller.observe(ctx, elapsed, append(extraLabels, stepType, string(attr.GetOperation()), strconv.FormatBool(rejected))...)
 }
 
-// ObserveWebhook records admission related metrics for a admission webhook.
-func (m *AdmissionMetrics) ObserveWebhook(ctx context.Context, name string, elapsed time.Duration, rejected bool, attr admission.Attributes, stepType string, code int) {
+// ObserveWebhook records admission related metrics for an admission webhook.
+func (m *AdmissionMetrics) ObserveWebhook(ctx context.Context, filter LabelCardinalityFilter, name string, elapsed time.Duration, rejected bool, attr admission.Attributes, stepType string, code int) {
 	// We truncate codes greater than 600 to keep the cardinality bounded.
 	if code > 600 {
 		code = 600
 	}
 	m.webhookRequest.WithContext(ctx).WithLabelValues(name, stepType, string(attr.GetOperation()), strconv.Itoa(code), strconv.FormatBool(rejected)).Inc()
 	m.webhook.observe(ctx, elapsed, name, stepType, string(attr.GetOperation()), strconv.FormatBool(rejected))
+	m.webhookSLO.WithContext(ctx).WithLabelValues(webhookSLOMetricLabels(name, attr, filter)...).Observe(elapsed.Seconds())
+}
+
+func webhookSLOMetricLabels(name string, attr admission.Attributes, filter LabelCardinalityFilter) []string {
+	labels := []string{name, string(attr.GetOperation()), "", "", "", ""}
```

Comment:
instead of `""` should we use `"other"` to denote label values that we have not allowlisted?

Reply:
I think that makes sense :+1:

Reply:
I used `OTHER` given that `other` is a valid resource or group name.